libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -2174,11 +2199,53 @@ function getStringifiedValueFromEditorViewModel(parameter: ParameterInfo, isDefi
       return editorOptions?.isOldFormat
         ? iterateSimpleQueryBuilderEditor(editorViewModel.itemValue, editorViewModel.isRowFormat)
         : JSON.stringify(recurseSerializeCondition(parameter, editorViewModel.items, isDefinitionValue));
+    case constants.EDITOR.FLOATINGACTIONMENU:
+      if (!editorViewModel || editorOptions?.menuKind !== FloatingActionMenuKind.outputs) {
+        return undefined;
+      }
+
+      return getStringifiedValueFromFloatingActionMenuOutputsViewModel(parameter, isDefinitionValue, editorViewModel);
     default:
       return undefined;
   }
 }
 
+const getStringifiedValueFromFloatingActionMenuOutputsViewModel = (
+  parameter: ParameterInfo,
+  isDefinitionValue: boolean,
+  editorViewModel: FloatingActionMenuOutputViewModel
+): string | undefined => {
+  const value: typeof editorViewModel.schema & { additionalProperties?: { outputValueMap?: Record<string, unknown> } } = clone(
+    editorViewModel.schema
+  );
+  const schemaProperties: typeof editorViewModel.schema.properties = {};
+  const outputValueMap: Record<string, unknown> = {};
+
+  // We want to cast (for example) "1" to 1 if the dynamically added parameter type is 'Number'
+  const commonProperties = { supressCasting: false, info: parameter.info };
```

Comment:
Doesn't seem like this does what I want ("1" to 1), searching for a helper that will do this cast....

Reply:
using `parameterValueWithoutCasting()`